### PR TITLE
ATT&CK Navigator connector plugin

### PR DESCRIPTION
**Pull request by bfuzzy1**
A simple plugin that adds ATT&CK Navigator to the Operator platform. Internet access is required for this plugin to work properly.

### CODE REVIEW

**Review by privateducky — CHANGES_REQUESTED** [review]
This is great - I love how lightweight this is. Can you rename the plugin "Navigator" so the file path will be: plugins/navigator/Navigator.html

I could see the space/copyright symbol being problematic on some folks machines.

By the way, one of our guys may add a full-screen option for plugins, which will allow this to get maximum space!

**Comment by bfuzzy1** [comment]
> This is great - I love how lightweight this is. Can you rename the plugin "Navigator" so the file path will be: plugins/navigator/Navigator.html
> 
> I could see the space/copyright symbol being problematic on some folks machines.
> 
> By the way, one of our guys may add a full-screen option for plugins, which will allow this to get maximum space!

Good call out on that symbol! Everything should be good to go now. Having full-screen would be awesome!

File: plugins/navigator/Navigator.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div>
+<div class="profile-heading-container">
+    <div class="body">
+        <strong class="profile-heading">Navigator</strong>
+        <p>This plugin acts as a connector to <a href="https://github.com/mitre-attack/attack-navigator">MITRE's ATT&CK Navigator</a>. 
+
+        The principal feature of the Navigator is the ability for users to define layers - custom views of the ATT&CK knowledge base - e.g. showing just those techniques for a particular platform or highlighting techniques a specific adversary has been known to use. Layers can be created interactively within the Navigator or generated programmatically and then visualized via the Navigator. </p>
+    </div>
+</div> 
+    <object type="text/html" data="https://mitre-attack.github.io/attack-navigator/"  width="1200px" height="800px"style="overflow:auto;background:white">
```
Comment (privateducky) [inline plugins/navigator/Navigator.html:10]: @lzimm any quick tips to make this resizable with the window?

File: plugins/navigator/Navigator.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div>
```
Comment (privateducky) [inline plugins/navigator/Navigator.html:1]: one minor thing: looks like an orphan div here, you can delete that

**Comment by privateducky** [comment]
I've been toying w/this for a few min. Easy interaction. I could see a future version allowing you to load navigator layers of your adversaries no problem. We'll get this merged in pretty soon, just want to see if we can auto-size the div to the window by chance.

File: plugins/navigator/Navigator.html (hunk 2 above)
Comment (bfuzzy1) [reply plugins/navigator/Navigator.html]: Removed that in this [commit](https://github.com/preludeorg/operator-support/pull/33/commits/26b56b2ee8f159681b59d2058cf416f94b9a3a9f)

File: plugins/navigator/Navigator.html (hunk 1 above)
Comment (bfuzzy1) [reply plugins/navigator/Navigator.html]: Fixed that in this [commit](https://github.com/preludeorg/operator-support/pull/33/commits/26b56b2ee8f159681b59d2058cf416f94b9a3a9f)

**Comment by bfuzzy1** [comment]
@privateducky Everything should be fixed now!

**Review by privateducky — APPROVED** [review]
great work!

**Pull request merged**